FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -458,7 +464,15 @@ - (void)URLSession:(NSURLSession *)session
                                                              error:&dataError];
   NSString *targetTemplateVersion = _configFetch.templateVersionNumber;
   if (dataError == nil) {
-    targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    if ([response objectForKey:kTemplateVersionNumberKey]) {
+      targetTemplateVersion = [response objectForKey:kTemplateVersionNumberKey];
+    }
+    if ([response objectForKey:kIsFeatureDisabled]) {
+      gIsFeatureDisabled = [response objectForKey:kIsFeatureDisabled];
+      if (gIsFeatureDisabled) {
+        [self pauseRealtimeStream];
```

Comment:
do we still want to call autoFetch after pausing the stream? i thought featureDisabled would turn off realtime entirely

Reply:
Good catch, added a check to see of flag is enabled before autofetching